pal/input/native/mouse-input.ts

```diff
@@ -95,9 +99,14 @@ export class MouseInputSource {
             const eventMouse = new EventMouse(eventType, false, this._preMousePos, mouseEvent.windowId);
             eventMouse.setLocation(location.x, location.y);
             eventMouse.setButton(button);
-            eventMouse.movementX = location.x - this._preMousePos.x;
-            eventMouse.movementY = this._preMousePos.y - location.y;
-
+            if (this._pointLocked && eventType === InputEventType.MOUSE_MOVE) {
```

Comment:
should we ignore the other mouse event, like mouse down, mouse up ?

Reply:
Other events do not need to use delta directly.